tests/gtests/CMakeLists.txt

```diff
@@ -25,6 +25,11 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}
                     ${CMAKE_CURRENT_SOURCE_DIR}/../../src/common
                     ${CMAKE_CURRENT_SOURCE_DIR}/../../src/cpu
                     )
+if(MKLDNN_DISABLE_JIT)
```

Comment:
These should propagated from the top-level CMakLists.txt

Reply:
Will change it in the next pull request.